spec/std/file_spec.cr

```diff
@@ -741,6 +741,17 @@ describe "File" do
     end
   end
 
+  it "reads at offset" do
+    filename = "#{__DIR__}/data/test_file.txt"
+    file = File.open(filename)
+    file.read_at(6, 100) do |io|
+      io.gets_to_end.should eq("World\nHello World\nHello World\nHello World\nHello World\nHello World\nHello World\nHello World\nHello Worl")
+    end
+    file.read_at(0, 240) do |io|
+      io.gets_to_end.should eq(File.read(filename))
+    end
+  end
+
```

Comment:
I'd add a spec on the invalid arguments cases: negative offset and offset plus size out of bounds